proxy/http/Http1Transaction.cc

```diff
@@ -78,15 +73,25 @@ Http1Transaction::allow_half_open() const
 }
 
 void
-Http1Transaction::increment_client_transactions_stat()
+Http1Transaction::increment_txn_stat()
```

Comment:
I'm not too sure if we should do this renaming, because it's unclear what stat is going to be incremented. Using this for the both client side and server side is ok, but the name should say that it increments a stat for number of transactions.

Reply:
I'll make note of that. If this goes the way I want, it will just be moved into the constructor.